src/components/PlioQuestion.vue

```diff
@@ -218,13 +227,88 @@ export default {
           this.hideSkipButton();
           this.isAnswerSubmitted = true;
           this.disableRadioButtons();
-        }, 200);
+        });
       }
 
       this.text = "";
       this.show = true;
       document.querySelector("body").classList.add("overflow-hidden");
       this.newProgressBarInfo['progressPercent'] = this.progressBarInfo['progressPercent']
+      this.handleImage()
+      this.styleQuestion()
+      this.styleOptions()
+      this.renderContent()
+    },
+
+    handleImage(){
+      this.$nextTick(() => {
+        // get question element
+        var question = document.getElementById('question')
+        if (question == null) return
+        // extract all <p> tags from the question element
+        var p_tags = question.querySelectorAll('p')
+        if (p_tags == null || p_tags.length == 0) return
+        // find which <p> tag has an <img> tag, cut it and
+        // paste it right outside, just below it.
+        p_tags.forEach(p_tag => {
+          var img_tag = p_tag.querySelector('img')
+          if (img_tag != null) {
+            p_tag.removeChild(img_tag)
+            p_tag.parentNode.insertBefore(img_tag, p_tag.nextSibling)
+            img_tag.setAttribute("id", "question_image")
+            // scale the image such that width is 1/3rd of screen width
+            // maintaining the aspect ratio
+            var heightToWidth = (img_tag.offsetHeight/img_tag.offsetWidth)
+            img_tag.style.width = String(window.screen.availWidth/3) + 'px'
+            img_tag.style.height = String(img_tag.offsetWidth * heightToWidth) + 'px'
+          }
+        })
+      })
+    },
+
+    renderContent(){
+      // force rendering of question and option texts
+      this.$nextTick(() => {
+        MathJax.Hub.Queue(["Typeset",MathJax.Hub,"question"]);
+        MathJax.Hub.Queue(["Typeset",MathJax.Hub,"option"]);
+      })
+    },
+
+    styleQuestion(){
+      // for each <p> tag in the question, set some CSS
+      // specifically overrides default <p> CSS for these tags
+      this.$nextTick(() => {
+        var question = document.getElementById('question')
+        if (question == null) return
+  
+        var p_tags = question.querySelectorAll('p')
+        if (p_tags == null || p_tags.length == 0) return
+
+        p_tags.forEach(p_tag => {
+          p_tag.style.lineHeight = '100%'
+          p_tag.style.marginRight = 'auto'
+          p_tag.style.marginBlock = '5px';
+        })
```

Comment:
The styling here is done in JS because the `<p>` tags by default have a styling (provided by the browser). It was not possible to override the default styles in CSS because the `<p>` tags are created only afterwards.

Reply:
not sure if @pritamps knows a better way of doing it?

Reply:
I'm not sure what @deepansh96 is saying is true. if the styles are defined and CSS is ready, they should be loaded whenever the paragraphs are loaded. For example, you're not setting fonts etc here, and they are being read from CSS only.

what I would suggest here is to add a class to each of the `p_tag`s, and define that class's style in the CSS

Reply:
I tried this. Added classes to each of the `<p>` tags and defined those CSS values in that class.
It didn't work somehow.
While inspecting the rendered HTML code, I was able to see that the class was added to the rendered `<p>` tags but the CSS styles were not applied. The CSS styles that were applied to those `<p>` tags were the default chrome ones.